queries.sql

```diff
@@ -1,3 +1,8 @@
-/*Queries that provide answers to the questions from all projects.*/
-
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
[OPTIONAL]

Hi, thank you for your submission. I have reviewed your code and found that it meets the project requirements. You have created queries to answer the questions using the correct syntax and logic. Well done!

However, I have some optional suggestions to improve your code further:

- You should use the BETWEEN operator instead of the >= and <= operators when comparing a value to a range. This can make your code more concise and readable. For example, instead of writing `date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12', `you should write `date_of_birth BETWEEN '2016/01/01' AND '2019/31/12`'.
- You should use the IN operator instead of the OR operator when checking if a value matches any of a list of values. This can also make your code more concise and readable. For example, instead of writing `name = 'Pikachu' OR name = 'Agumon'`, you should write `name IN ('Pikachu', 'Agumon')`.

These suggestions are not mandatory, but they are good practices to follow when writing SQL queries. They can help you avoid potential errors and improve the readability and performance of your code.

I have provided an improved version of your code below. Please review it and consider making the suggested changes. If you have any questions or doubts, please let me know.